### DISCUSSION TITLE
What to send through variable_stars tutorial notebook

### DISCUSSION THREAD

**Discussion by jsv1206** [lsst-epo/citizen-science-notebooks#69] [General]
- send light curve plots and image cutouts
- send flux catalog and image cutouts where Zooniverse can make the light curve from the text file

**Comment by jsv1206** [comment]
- gri image
- single band image

**Comment by jsv1206** [comment]
@ericdrosas87 Do you think it is possible to add the infrastructure to send flipbook images and the associates light curve png plots simultaneously?

**Comment by beckynevin** [comment]
I think we need to have a full group discussion about this.

**Comment by beckynevin** [comment]
We've had several discussions and concluded that the variable_stars notebook (now named Send_Flipbook...) will simply send a flipbook of images. I'm working on a separate notebook to demonstrate sending tabular data that will use Zooniverse's existing widget for making light curves.
